runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1650,6 +1695,11 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 				enableOriginalJDK8HeapSizeCompatibilityOption = true;
 			}
 		}
+		IDATA testContainerMemLimitEnabled = FIND_AND_CONSUME_VMARG(EXACT_MATCH, "-XX:+fvtest_testContainerMemLimit", NULL);
```

Comment:
all fvtest_ options are -XXgc: style and let's keep this one there too

we'll lose 'disable' ability, but that's fine

Reply:
This is because XXgc is parsed after the computeDefaultMaxHeap(), and that would be too late for this case. Maybe a better name without fvtest?

Reply:
we  don't need this for restore side (restore will inherit it from snapshot), so that's why you did not put it in 'reconfigurable' options?

Reply:
Yes

Reply:
agreed to keep it as-is, since -XXgc: options are parsed after this